wp-content/themes/access/views/screener-field-partials/question-toggles-unemployed.twig

```diff
@@ -1,11 +1,11 @@
-<div class="c-question">
+<div class="c-question"  aria-live="polite">
```

Comment:
@theasteve It looks like these attributes snuck back into the templates, we'll want to remove them.